src/main/java/uk/gov/homeoffice/digital/sas/person/model/Person.java

```diff
@@ -18,19 +18,21 @@
 import org.hibernate.validator.constraints.Length;
 import uk.gov.homeoffice.digital.sas.jparest.annotation.Resource;
 import uk.gov.homeoffice.digital.sas.jparest.models.BaseEntity;
+import uk.gov.homeoffice.digital.sas.kafka.listener.KafkaEntityListener;
+import uk.gov.homeoffice.digital.sas.kafka.message.Messageable;
 import uk.gov.homeoffice.digital.sas.person.enums.TermsAndConditions;
-import uk.gov.homeoffice.digital.sas.person.listeners.PersonKafkaEntityListener;
 
 @Resource(path = "persons")
 @Entity(name = "person")
 @JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
-@EntityListeners(PersonKafkaEntityListener.class)
+@EntityListeners(KafkaEntityListener.class)
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
 @Getter
 @Setter
-public class Person extends BaseEntity {
+@SuppressWarnings("squid:S2160") //Suppressed overriding equals until eahw-2652
```

Comment:
I think it would be better not to suppress this warning so that it comes up in Sonar analysis. This will serve as a reminder

Reply:
I agree. I would prefer not to suppress but Sonar is reporting a C rating and A is required.

Reply:
This is puzzling. As far as I am aware, Sonar scan didn't fail before for the Person class not overriding equals(). Could this be a different issue. I'd be curious to see the issue.